<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>forceSenseControllerClass</class>
 <widget class="QMainWindow" name="forceSenseControllerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>forceSenseController</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1076</width>
         <height>2000</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>2000</height>
        </size>
       </property>
       <widget class="QPushButton" name="readForceAndShowButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>10</y>
          <width>161</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>读取并显示六维力</string>
        </property>
       </widget>
       <widget class="QPushButton" name="zeroSenseButton">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>10</y>
          <width>111</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>传感器清零</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>310</y>
          <width>1011</width>
          <height>501</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,1,5,1,5">
           <item>
            <widget class="QChartView" name="Fx_show" native="true"/>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Preferred</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QChartView" name="Fy_show" native="true"/>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Preferred</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QChartView" name="Fz_show" native="true"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,1,5,1,5">
           <item>
            <widget class="QChartView" name="Mx_show" native="true"/>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Preferred</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QChartView" name="My_show" native="true"/>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Preferred</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QChartView" name="Mz_show" native="true"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>691</width>
          <height>120</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_26">
           <property name="text">
            <string>二次重力补偿相关(含'代表示教器示数)：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1,1,1,1">
           <item>
            <widget class="QPushButton" name="get_once_Button">
             <property name="text">
              <string>获取当前位姿力信息</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="caculate_Button">
             <property name="text">
              <string>计算真实重力</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clear_Button">
             <property name="text">
              <string>清空当前位姿数</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>当前已有位姿数:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="curNum_lineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,5,1,5,1,5,1,5">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>G':</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="G_robot_lineEdit">
             <property name="text">
              <string>6.6708</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>d'x:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="dx_robot_lineEdit">
             <property name="text">
              <string>-0.002</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>d'y:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="dy_robot_lineEdit">
             <property name="text">
              <string>0.001</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>d'z:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="dz_robot_lineEdit">
             <property name="text">
              <string>0.035</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,5,1,5,1,5,1,5">
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>G:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="G_real_lineEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>dx:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="dx_real_lineEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string>dy:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="dy_real_lineEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>dz:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="dz_real_lineEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>190</y>
          <width>721</width>
          <height>101</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_27">
           <property name="text">
            <string>相对于第五轴固定力矩补偿相关：</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1,1,0,1,1">
           <item>
            <widget class="QPushButton" name="get_Compensation_Button">
             <property name="text">
              <string>获取力/力距与转角</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="caculate_Compensation_Button">
             <property name="text">
              <string>计算补偿力矩</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="clear_Compensation_Button">
             <property name="text">
              <string>清空当前位姿数</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cancel_Compensation_Button">
             <property name="text">
              <string>取消补偿</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>当前已有位姿数:</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="curNum_Compensation_lineEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,5,1,5,1,5,0,0,0,0,0,0">
           <item>
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string>M_e_x5:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="M_e_x5_lineEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>M_e_y5:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="M_e_y5_lineEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>M_e_z5:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="M_e_z5_lineEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_21">
             <property name="text">
              <string>M_x0:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="M_x0_lineEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_23">
             <property name="text">
              <string>M_y0:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="M_y0_lineEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="label_22">
             <property name="text">
              <string>M_z0:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="M_z0_lineEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
